kuma/static/styles/components/payments/banner.scss

```diff
@@ -194,6 +199,7 @@
     }
 
     .payments-form {
+        @include bidi(((float, right, left),));
```

Comment:
It looks like this change affects the LTR layout as well as the RTL, right? Just want to make sure it is an intentional change, not an accidental one.